zmon_aws_agent/aws.py

```diff
@@ -295,9 +295,14 @@ def get_running_apps(region, existing_entities=None):
                     if 'Name' in tags:
                         ins['name'] = tags['Name'].replace(' ', '-')
 
-            if 'application_id' in ins and not (now.minute % 10):
-                ins['events'] = get_instance_events(aws_client, i)
-                ins['block_devices'] = get_instance_devices(aws_client, i)
+            if 'application_id' in ins:
+                if not (now.minute % 10):
+                    ins['events'] = get_instance_events(aws_client, i)
+                    ins['block_devices'] = get_instance_devices(aws_client, i)
+                else:
```

Comment:
What about `block_devices`?  Shouldn't we copy those as well?

Reply:
They're filled (line 244) and not cleared explicitly like the events (line 255)

Reply:
Doesn't that mean we get block devices two times per instance if it happens to run on a round 10-minute number?

Reply:
OK, I think this can be fixed separately.